docs/pages/Cardinal/CoreConcepts/World.mdx

```diff
@@ -0,0 +1,141 @@
+# World
```

Comment:
In general, I think for Components/Reads/Systems/Transactions/World we should follow the docs pattern that https://nextjs.org/docs/app/api-reference/components/image has

1. Provide a full example of how to define/use it
2. Provide a table with the parameters, types, and description